### ISSUE TITLE
option "directions" expects STDIN not [FILE]

### ISSUE THREAD

**Issue by mengwong** [legalese/commonform-cli#5]
```20170720-14:03:53 mengwong@venice2:~/non-db-src/l/commonform-cli% bin/commonform directions ../commonform-agreement-starter/agreement.commonform```
doesn't work

```20170720-14:03:59 mengwong@venice2:~/non-db-src/l/commonform-cli% bin/commonform directions < ../commonform-agreement-starter/agreement.commonform```
does:
```
[{"identifier":"Effective Date","path":["content",1]},{"identifier":"First Party Name","path":["content",3]},{"identifier":"First Party Jurisdiction","path":["content",5]},{"identifier":"First Party Form","path":["content",7]},{"identifier":"Second Party Name","path":["content",11]},{"identifier":"Second Party Jurisdiction","path":["content",13]},{"identifier":"Second Party Form","path":["content",15]}]20170720-14:04:01 mengwong@venice2:~/non-db-src/l/commonform-cli% 
```

**Comment by mengwong** [comment]
we will solve this by leaving `directions` behaviour untouched and adding `blanks`.

**Comment by mengwong** [comment]
resolved by 65ac0d73e92382c24c0c7e5650234ef8445a9924
